site/docs/v1/tech/tutorials/start-and-stop-pre-1-37.adoc

```diff
@@ -0,0 +1,132 @@
+---
+layout: doc
+title: Start and Stop FusionAuth
+description: Learn how to start and stop the FusionAuth services
+navcategory: getting-started
+---
+
+== Start and Stop FusionAuth
+
+:137_link: /docs/v1/tech/tutorials/start-and-stop
+[WARNING.warning]
+====
+include::docs/v1/tech/shared/_pre-1-37-warning.adoc[]
+====
+
+If you are using the FusionAuth server version, may need to manually start or stop FusionAuth periodically. If you find yourself in this situation you'll be glad you know how.
```

Comment:
Is this the copy we use other places? `server version` may be confusing. Should we say 

> If you are self-hosting FusionAuth, you may need...